frontend/src/Commons/MainPageHeader/MainPageHeader.styles.ts

```diff
@@ -0,0 +1,66 @@
+import styled from '@emotion/styled';
+import * as tokens from '@bcgov/design-tokens/js';
```

Comment:
https://webpack.js.org/guides/tree-shaking/

Reply:
So just limit what we only need instead of using '*'?

Reply:
That's the jist of it, yes :) 
It won't really affect performance, but reduces bundle size. This is a small example, but always a good thing to learn early

Reply:
Gotcha!